install/kubernetes/cilium/values.yaml

```diff
@@ -277,6 +277,9 @@ global:
     # mode is the mode of NodePort feature
     mode: "hybrid"
 
+    # acceleration is the option to accelerate NodePort via XDP
+    acceleration: "none"
```

Comment:
Is there a negative to configuring `generic` by default, or are you looking to gradually phase it in with beta in v1.8 and apply by default in v1.9? Seems like initial XDP was merged in Linux v4.8, but I don't know if we rely on additional features that bump the minimum kernel higher than that for this feature.

Reply:
(I see after this you plan to later merge into the probe mode).

Reply:
Addendum: If we don't support this on all kernels, we should extend this doc:

https://docs.cilium.io/en/v1.7/install/system_requirements/#advanced-features-and-required-kernel-version

Reply:
I plan to later use generic mode for our CI. But in general, I would like users to avoid using generic mode if possible and only for testing since it can become quite slow as it will linearize every skb and turn off GRO. Perhaps the `generic` option should even be hidden from the help and documentation. 'officially'. What people should use is the native one whenever they have the proper driver for it.